### DISCUSSION TITLE
Host in subfolder

### DISCUSSION THREAD

**Discussion by fritterhoff** [gitpod-io/openvscode-server#203] [Q&A] [answered]
Due to some internal technical limitations I want to host openvscode-server behind a reverse proxy (in detail caddyserver) in a subdirectory/path instead of the root path. 
Hosting in the root path works quite well (beside the bug with firefox and the csp errors), using a subdirectory/path does not even load a single pixel ... 

Any idea what I can do?

**Comment by akosyakov** [comment, marked as answer]
Yes there are, please see: https://github.com/gitpod-io/openvscode-server/issues/90 It should be allowed in the upstream first.
